examples/from_excel.py

```diff
@@ -0,0 +1,19 @@
+"""Example of using PandasAI with am Excel file."""
+
+import pandas as pd
+
+from pandasai import PandasAI
+from pandasai.helpers.from_excel import from_excel
+from pandasai.llm.openai import OpenAI
+
+# convert csv to xlsx for this example
+df = pd.read_csv("examples/data/Loan payments data.csv")
+df.to_excel("examples/data/Loan payments data.xlsx", index=False)
```

Comment:
I would say let's rather add a `.xlsx` file in the `/data` folder and include that one. All these transformation make the example less intuiteve!